### PR TITLE
SW-1799 Country/subdivision dropdown fixes

### PR DESCRIPTION
**Pull request by karthikbtf**
- switch country and subdivision ordering
- fix edit view to populate country (clearing of fields is broken, filed another ticket for that)
- fix edit view to populate GPS entries
- fix detail panel to show user friendly country/subdivision names

### CODE REVIEW

File: src/components/accession2/view/DetailPanel.tsx [hunk 1]
```diff
@@ -61,6 +75,19 @@ export default function DetailPanel(props: DetailPanelProps): JSX.Element {
...
   };
 
+  const getCollectionSiteAddress = () => {
+    const city = accession?.collectionSiteCity;
+    const country = accession?.collectionSiteCountryCode;
+    const subdivision = accession?.collectionSiteCountrySubdivision;
+    const data = [
+      city,
+      countries && country && subdivision ? getSubdivisionByCode(countries, country, subdivision)?.name : '',
+      countries && country ? getCountryByCode(countries, country)?.name : '',
```
Comment (constanzauanini) [inline src/components/accession2/view/DetailPanel.tsx:85]: I don't think we need to show the subdivision and country names here. In figma, the example shows CA, USA. Maybe confirm with Ming?

Comment (karthikbtf) [reply src/components/accession2/view/DetailPanel.tsx:85]: I don’t think she was being specific. Doesn’t make sense to show something else that user didn’t see when selecting. The codes are not understood by all. I can remove this for now and create a new bug for Allison and Ming to follow up on.

Comment (constanzauanini) [reply src/components/accession2/view/DetailPanel.tsx:85]: Yes, reasonable!

**Review by constanzauanini — APPROVED** [review]

File: src/components/accession2/view/DetailPanel.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/accession2/view/DetailPanel.tsx:85]: @constanzauanini I will get this in for now - if feedback on https://terraformation.atlassian.net/browse/SW-1822 is different I will follow up accordingly.

**Pull request merged**